cli_reference/opm/cli-opm-ref.adoc

```diff
@@ -0,0 +1,158 @@
+[id="cli-opm-ref"]
+= opm CLI reference
```

Comment:
Looks like the examples here contain auto-generated text, is that correct? Do you want me to review them?

Also some of the other CLI references include an example title before the code boxes, e.g., .Example usage; is that something you'd like to include here too?

Reply:
Ended up reworking these to match the style in the Operator SDK CLI reference.